course/meetings/eeb-marty-2017-04-13.md

```diff
@@ -0,0 +1,59 @@
+	- Some ecology tidbits (mostly for Madeleine's reference):
+		- they discuss in 3rd year logistic growth, the Allee effect, the spruce bugworm model circa 1970s (dN/dt = G(N)-H(N)), G(N) is logistic growth, H(N) is type III predator functional response
+		- functional responses: type I is linear, type II is saturating (Michaelis-Menten type), type III is sigmoidal. Illustrate types with analogy of you going to an all-you-can-eat buffet - what does your response look like? Plots are rate of eating of 1 predator vs. prey abundance. 
+	
+- Course structure and logistics:
+	- Martin will **not** give any lectures, he is only there to help with admin stuff. 
+	- Room has been booked: it's the CQuest (?) room, a computer lab, for both days. We can go and check it out at some point. It has a projector and screen.
+	- We will communicate with IT to discuss the software the computers need: R, RStudio, Git, R packages? Saving permissions?
+	- We can bring in guest lecturers if we want
+	- Unstructured project work time is an acceptable thing to have in place of lecture later in the course.
+	- We can monitor enrolment over the summer, but it won't be a good indicator really until a few weeks into the course. 
+	- Avoid a course website that isn't Blackboard if possible - best to do all dissemination of materials and communication on there unless it's way too clunky for something.
+	- Gather student feedback early and often - we can't change the assessment scheme once it's set, but we can change the content and length of assignments. 
+	- We can't make any changes to the syllabus after we hand it out to students at the beginning of the course and give a copy to the department. 
+	- Other than syllabus and getting paid (Asher is contact for that), there isn't any paperwork we need to do.
+
+- Student projects and data:
+	- Martin has lots of salmon host-parasite timeseries data, but he suggested it might be better to get data from online repositories so students get experience doing that. Some suggested online sources:
+		- Ecology Journal has published databases
+		- [Global population dynamics database](https://www.imperial.ac.uk/cpb/gpdd2/secure/login.aspx) - timeseries data, appears you need a login
+		- [RAM legacy database](http://ramlegacy.marinebiodiversity.ca/ram-legacy-stock-assessment-database) - marine fisheries data
+	- No opinion from Martin on using strictly timeseries data vs. being flexible - depends on us and students
+	- Students will need some guidance for what kinds of questions & analysis they might want to do - up to us to keep it simple. It's likely they'll be out of their depth too fast otherwise. 
+	- Projects might not necessarily involve data - they could just be simulations. I'm okay with this, any comments?
```

Comment:
I'm also ok with this, but I think I would prefer if they find a data set. Partly because of what you mentioned above, that the students should get experience finding data sets online, and also because I think there are more things that could go wrong if they are to simulate their data and not just use what is already there... but it might be fine if the student(s) already has some experience in this realm.